pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/tasks/action/CreateVirtualEnvAction.java

```diff
@@ -76,9 +76,12 @@ public void buildVenv(@Nullable Consumer<File> customize) {
         OutputStream outputStream = new ByteArrayOutputStream();
         ExecResult execResult = project.exec(execSpec -> {
             container.setOutputs(execSpec);
+            // For virtualenv >= 16.1
+            // execSpec.environment("PYTHONPATH", new File(packageDir, "src"));
+            execSpec.environment("PYTHONPATH", packageDir);
             execSpec.commandLine(
                 pythonDetails.getSystemPythonInterpreter(),
-                new File(packageDir, "virtualenv.py"),
```

Comment:
Let's keep the original line.
In fact, all the changes in this file can be reverted and we'll think about the way to accomplish this going forward when we can actually move to virtualenv>16.0.0 and wheel > 0.31.0. We cannot for now because of pex limitations and this code is stable as is.

Reply:
Okay, for now.  See comments above for I think a better long term approach.